assets/js/admin.js

```diff
@@ -178,3 +178,29 @@ jQuery(document).ready(function($) {
 		$('#in-' + taxonomy + '-' + id + ', #in-popular-' + taxonomy + '-' + id).prop( 'checked', c );
 	});
 });
+
+// Select all elements with the class "promote_job"
+const promote_job = document.querySelectorAll('.promote_job');
+const promoteDialog = document.getElementById('promoteDialog');
+
+promote_job.forEach(function (element) {
+		element.addEventListener('click', function ( event ) {
+			event.preventDefault();
+			promoteDialog.showModal();
+			let promoteID = event.target.dataset.post;
+			let promote_job_dialog = document.querySelector('.promote-buttons-group .button-primary');
+			promote_job_dialog.setAttribute('href', promoteID );
+		});
+});
+
+customElements.define('promote-job-modal',
+  class extends HTMLElement {
```

Comment:
I imagine you used spaces instead of tabs accidentally here, right?

Reply:
😂   Sometimes I write things in a codepen or something and then move it over and forget to check for this stuff...